src/helpers/replaceWithCurrentDomain.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+export const replaceDomain = (link: string, domain: string) => {
+  const parsed = new URL(link);
```

Comment:
Does it work in IE? https://developer.mozilla.org/en-US/docs/Web/API/URL#Browser_compatibility

Reply:
Maybe we can polyfill it too. Or just revert to adding a dependency for this :joy: Or just using a regexp to implement a limited solution that nevertheless covers our needs.

Reply:
@git-toni what say you?

Reply:
Yep only Opera Mini is missing https://caniuse.com/#search=url shouldn't be an issue, should it?